doc-content/enterprise-only/installation/installer-run-cli-proc.adoc

```diff
@@ -96,8 +97,21 @@ Make a note of the user name and password. You will need them to access {CENTRAL
 ----
 Would you like to generate an automatic installation script and properties file?
 ----
-. Enter `y` to create an XML file that contains the installation data, or `n` to complete the installation. If you enter `y`, you are prompted to specify a path for the XML file.
+. Enter `y` to create XML files that contains the installation data, or `n` to complete the installation. If you enter `y`, you are prompted to specify a path for the XML files.
 . Enter a path or press the Enter key to accept the suggested path.
++
+The installer generates two files. The `auto.xml` file automates future installations and the `auto.xml.variables` file stores user passwords and other sensitive variables.
+Use the `auto.xml` file on multiple systems to easily repeat a {PRODUCT} installation on the same type of server with the same configuration as the original installation. If necessary, update the `installpath` parameter in the `auto.xml` file. Enter the following command to perform an installation with the XML file:
++
+[source,subs="attributes+"]
+----
+ifdef::PAM[]
+java -jar {PRODUCT_INIT}-installer-{PRODUCT_VERSION_LONG}.jar <path-to-auto.xml-file>
+endif::PAM[]
+ifdef::DM[]
+java -jar {PRODUCT_INIT}-installer-{PRODUCT_VERSION_LONG}.jar <path-to-auto.xml-file>
+endif::DM[]
```

Comment:
Just curious why the ifdefs. The snippets are identical, and the variables will be handled for the respective product.

Reply:
Fixed. Reason is most likely historic - was conditionalized before we had variables.